### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by LovelyBuggies**
Fix some bugs in example codes and docs.

### CODE REVIEW

**Comment by henryiii** [comment]
Can you rebase on or merge with latest develop, please? Feel free to drop the change I made if you rebase, as I've made it in the fix I applied to develop.

File: setup.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def build_extensions(self):
 extras = {
     "test": ["pytest", "pytest-benchmark", "numpy", 'futures; python_version < "3"'],
     "docs": ["Sphinx>=2.0.0", "recommonmark>=0.5.0", "sphinx_rtd_theme"],
-    "examples": ["numpy", "ipykernel", "matplotlib"],
+    "examples": ["numpy", "ipykernel", "matplotlib", "xarray"],
```
Comment (henryiii) [inline setup.py]:
```suggestion
    "examples": ["numpy", "ipykernel", "matplotlib", "xarray", "xhistogram"],
```

**Comment by henryiii** [comment]
At the moment, I'm just adding the missing required packages one by one as they show up as ImportErrors in the tests.

**Comment by henryiii** [comment]
🎉 ! Thank you so much!

**Comment by LovelyBuggies** [comment]
Sorry, I just see the message!

So what do I need to do if I want to pull more requests? Just rebase is ok?

**Comment by henryiii** [comment]
Assuming `origin` is the scikit-hep/boost-histogram and `LovelyBuggies` is your remote: `git fetch origin` followed by `git reset --hard origin/develop`. This will force your local checked-out develop branch to match the origin repository. Then just do `git push LovelyBuggies develop -f` to sync up your online repository.

If you prefer `origin` to be your remote, and `upstream` to the main repository, just change the names in the paragraph above.

Or you can just delete your fork then re-fork and re-clone, of course.

**Comment by LovelyBuggies** [comment]
You have just said a mouthful! Actually, I'm using your re-fork method 😅. But I will try the git method to develop in a better manner.

**Pull request merged**